lib/config.yml

```diff
@@ -26,6 +26,48 @@ routingTypes:
 # Show/hide elevation profile chart for walk/bike legs
 elevationProfile: true
 
+# Provide a list of operators to enhance the user experience for multi-agency
+# implementations. The order of the agencies will dictate the order that routes
+# appear in the route viewer. Also, optionally provide operator logos to appear
+# in narrative itinerary (matched on agency_id). Note: these logos will
+# override the values found in agency.txt#agency_branding_url.
+operators:
+  - id: psc
+    logo: https://d2tyb7byn1fef9.cloudfront.net/psc.png
+  - id: TRIMET
+    name: TriMet
+    logo: http://news.trimet.org/wordpress/wp-content/uploads/2019/04/TriMet-logo-300x300.png
+  - id: TRAM
+    logo: https://d2tyb7byn1fef9.cloudfront.net/tram.png
+  - id: CTRAN
+    name: C-TRAN
+    logo: https://d2tyb7byn1fef9.cloudfront.net/ctran.png
+    order: 9999999
+  - id: '1' # accidentally forgot to add C-TRAN agency_id to routes.
+    name: C-TRAN
+    logo: https://d2tyb7byn1fef9.cloudfront.net/ctran.png
+    order: 9999999
+  - id: '2' # accidentally forgot to add C-TRAN agency_id to routes.
```

Comment:
Why is C-TRAN in here 3 times?

Reply:
This was just to account for differences in GTFS between my local instance and the dev server. I think it should generally end up being `2` though, so I'll make that change.

Reply:
Actually, maybe we ought to just keep multiple for now so the deployment is not too fragile, I'll just add a clarifying comment.